wls-gui-wahllokalsystem/tests/stores/ereignisStore.spec.ts

```diff
@@ -656,3 +598,712 @@ function createUser(wahlbezirkID: string | undefined): User {
 
   return user;
 }
+
+function generateTestdataForAreKeineEreignisseFlagsValid() {
```

Comment:
interne funktionen mit unterstrich
```suggestion
function _generateTestdataForAreKeineEreignisseFlagsValid() {
```